components/SearchBar/SearchBar.tsx

```diff
@@ -40,6 +40,7 @@ const SearchBar = ({
   const [isExpanded, setIsExpanded] = useState(false);
   const [isWidthSet, setIsWidthSet] = useState(false);
   const [isSearchLoading, setIsSearchLoading] = useState(false);
+  const [isSearchInputFocusable, setIsSearchInputFocusable] = useState(false);
```

Comment:
What is the purpose of this state ("SearchInputFocusable")? further down I see the state being toggled but as far as I can see we don't do anything with it, it doesn't seem to have any effect ...

Reply:
Good call, I thought I needed it. But I don't!